pkgs/by-name/i3/i3pystatus/package.nix

```diff
@@ -0,0 +1,96 @@
+{
+  lib,
+  fetchFromGitHub,
+  libpulseaudio,
+  libnotify,
+  gobject-introspection,
+  python3Packages,
+  unstableGitUpdater,
+  fetchpatch,
+  extraLibs ? [ ],
+}:
+
+python3Packages.buildPythonApplication rec {
+  # i3pystatus moved to rolling release:
+  # https://github.com/enkore/i3pystatus/issues/584
+  version = "3.35-unstable-2024-06-13";
+  pname = "i3pystatus";
```

Comment:
Please set `pyproject = true;` and add `build-system = [ python3Packages.setuptools ];` (and whatever else needed to build).

Reply:
Yes, `checkInputs` is similar to `buildInputs` (e.g.: Python dependencies), while `nativeCheckInputs` is similar to `nativeBuildInputs` (e.g.: build dependencies).